### DISCUSSION TITLE
Do mantine have a way to modify the HTML document in next.js

### DISCUSSION THREAD

**Discussion by uimim** [mantinedev/mantine#977] [Feedback]
I want to add `lang="en"` to the `<html>`  in next.js . How can I do it?

as:https://nextjs.org/docs/advanced-features/custom-document

Have a good day :D

**Comment by rtivital** [comment]
https://mantine.dev/theming/next/#create-your-own-getinitialprops

**Reply by ShahriarKh** [reply]
The page is no longer available
